pulsar-client-cpp/pkg/rpm/docker-build-rpm.sh

```diff
@@ -20,7 +20,8 @@
 
 set -e
 
-ROOT_DIR=$(dirname $0)/../../../..
+ROOT_DIR=$(dirname $0)/../../..
+ROOT_DIR=$(realpath $ROOT_DIR)
```

Comment:
replace with this line (same fix as made in previous comment)
```
ROOT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )"/../../.. &> /dev/null && pwd )"
```

Reply:
good. fixed. thanks
it works